bsim module must be cloned for west to work

**Describe the bug**
When running west a prompt that looks like the following appears. 
`
FATAL ERROR: failed manifest import in bsim (tools/bsim)
  Failed importing "west.yml" from revision "908ffde6298a937c6549dbfa843a62caab26bfc5"
  Hint: for this to work:
          - bsim must be cloned
          - its manifest-rev ref must point to a commit with the import data
        To fix, run:
          west update
`
this seems to be a little backwards since it forces pulling in additional modules, outside of the minimum necessary to build.

**To Reproduce**
Steps to reproduce the behavior:
1. Follow the getting started guide but replace `west update` with `west update only_what_I_need`
2. Run a `west status`
You will get an error message that looks like the one above.
3. Run `west build -b qemu_x86 samples/hello_world/`, you get: `west: error: argument <command>: invalid choice: 'build'`
4. git revert 01a12f658 makes everything work again instantly (no need to re-init, update or anything)

**Expected behavior**
We should still be able to build without the bsim module. Or at least we should provide a work around to build without it.

**Environment (please complete the following information):**
 - OS: Seen on MacOS and Linux

Is there a workaround yet so I can test with the `3.4-RC2`?
I get this error:
```
CMake Error at /opt/zephyrproject-v3.4.0-rc2/zephyr/cmake/modules/zephyr_module.cmake:77 (message):
  ERROR: ManifestImportFailed: project <Project bsim
  ('/opt/zephyrproject-v3.4.0-rc2/tools/bsim') at
  908ffde6298a937c6549dbfa843a62caab26bfc5> value west.yml
```

Also since `3.4`, I have to call west with `sudo` to have the `build `option available.
(Note that I do not use `west.yaml` but use zephyr in `/opt`).

--- a/.github/workflows/bsim-tests.yaml
+++ b/.github/workflows/bsim-tests.yaml
@@ -116,7 +116,7 @@ jobs:
           || steps.check-networking-files.outputs.any_changed == 'true'
           || steps.check-common-files.outputs.any_changed == 'true'
         run: |
-          export BSIM_VERSION=$( west list bsim -f {sha} )
+          export BSIM_VERSION=$( west list bsim -f {revision} )
           echo "Manifest points to bsim sha $BSIM_VERSION"
           cd /opt/bsim_west/bsim
           git fetch -n origin ${BSIM_VERSION}

--- a/.github/workflows/doc-build.yml
+++ b/.github/workflows/doc-build.yml
@@ -69,7 +69,6 @@ jobs:
     - name: west setup
       run: |
         west init -l .
-        west update bsim
 
     - name: build-docs
       run: |
@@ -155,7 +154,6 @@ jobs:
     - name: west setup
       run: |
         west init -l .
-        west update bsim
 
     - name: build-docs
       run: |

--- a/west.yml
+++ b/west.yml
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: babblesim
+      url-base: https://github.com/BabbleSim
 
   group-filter: [-babblesim]
 
@@ -30,8 +32,86 @@ manifest:
     - name: bsim
       repo-path: babblesim-manifest
       revision: 908ffde6298a937c6549dbfa843a62caab26bfc5
-      import:
-        path-prefix: tools
+      path: tools/bsim
+      groups:
+        - babblesim
+    - name: babblesim_base
+      remote: babblesim
+      repo-path: base.git
+      path: tools/bsim/components
+      revision: 02838ca04c4562e68dc876196828d8121679e537
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_libPhyComv1
+      remote: babblesim
+      repo-path: ext_2G4_libPhyComv1.git
+      path: tools/bsim/components/ext_2G4_libPhyComv1
+      revision: 9018113a362fa6c9e8f4b9cab9e5a8f12cc46b94
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_phy_v1
+      remote: babblesim
+      repo-path: ext_2G4_phy_v1.git
+      path: tools/bsim/components/ext_2G4_phy_v1
+      revision: cf2d86e736efac4f12fad5093ed2da2c5b406156
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_channel_NtNcable
+      remote: babblesim
+      repo-path: ext_2G4_channel_NtNcable.git
+      path: tools/bsim/components/ext_2G4_channel_NtNcable
+      revision: 20a38c997f507b0aa53817aab3d73a462fff7af1
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_channel_multiatt
+      remote: babblesim
+      repo-path: ext_2G4_channel_multiatt.git
+      path: tools/bsim/components/ext_2G4_channel_multiatt
+      revision: e09bc2d14b1975f969ad19c6ed23eb20e5dc3d09
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_modem_magic
+      remote: babblesim
+      repo-path: ext_2G4_modem_magic.git
+      path: tools/bsim/components/ext_2G4_modem_magic
+      revision: cb70771794f0bf6f262aa474848611c68ae8f1ed
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_modem_BLE_simple
+      remote: babblesim
+      repo-path: ext_2G4_modem_BLE_simple.git
+      path: tools/bsim/components/ext_2G4_modem_BLE_simple
+      revision: ce975a3259fd0dd761d371b60435242d54794bad
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_device_burst_interferer
+      remote: babblesim
+      repo-path: ext_2G4_device_burst_interferer.git
+      path: tools/bsim/components/ext_2G4_device_burst_interferer
+      revision: 5b5339351d6e6a2368c686c734dc8b2fc65698fc
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_device_WLAN_actmod
+      remote: babblesim
+      repo-path: ext_2G4_device_WLAN_actmod.git
+      path: tools/bsim/components/ext_2G4_device_WLAN_actmod
+      revision: 9cb6d8e72695f6b785e57443f0629a18069d6ce4
+      groups:
+        - babblesim
+    - name: babblesim_ext_2G4_device_playback
+      remote: babblesim
+      repo-path: ext_2G4_device_playback.git
+      path: tools/bsim/components/ext_2G4_device_playback
+      revision: 85c645929cf1ce995d8537107d9dcbd12ed64036
+      groups:
+        - babblesim
+    - name: babblesim_ext_libCryptov1
+      remote: babblesim
+      repo-path: ext_libCryptov1.git
+      path: tools/bsim/components/ext_libCryptov1
+      revision: eed6d7038e839153e340bd333bc43541cb90ba64
+      groups:
+        - babblesim
     - name: canopennode
       revision: dec12fa3f0d790cafa8414a4c2930ea71ab72ffd
       path: modules/lib/canopennode

